ui/components/src/StatChart/StatChart.tsx

```diff
@@ -22,11 +22,14 @@ import { useChartsTheme } from '../context/ChartsThemeProvider';
 import { formatValue, UnitOptions } from '../model/units';
 import { EChart } from '../EChart';
 import { GraphSeries } from '../model/graph';
+import { useOptimalFontSize } from './calculateFontSize';
 
 use([EChartsLineChart, GridComponent, DatasetComponent, TitleComponent, TooltipComponent, CanvasRenderer]);
 
-const MIN_VALUE_SIZE = 12;
-const MAX_VALUE_SIZE = 36;
+const LINE_HEIGHT = 1.2;
```

Comment:
Is there any association between this constant and `DEFAULT_LINE_HEIGHT` in `calculateFontSize.ts`? If so, should one be set using the other?

Reply:
Actually yeah calculateFontSize is more accurate if using the line height from stat chart so updating this to removing DEFAULT_LINE_HEIGHT, thanks!